templates/terraform/examples/storage_hmac_key.tf.erb

```diff
@@ -0,0 +1,3 @@
+resource "google_storage_hmac_key" "<%= ctx[:primary_resource_id] %>" {
```

Comment:
Can you create an SA in this and interpolate the email as well? Since this generates a test, we need a real SA to get a key for.